derivation.nix

```diff
@@ -0,0 +1,83 @@
+# target-os = [ 'windows' 'linux' 'macos' 'android' 'ios' ]
+{ system ? builtins.currentSystem
+, config ? {}, overlays ? []
+, pkgs ? (import <nixpkgs> { inherit system config overlays; })
+, target-os ? "" }:
+
+with pkgs;
+  let
+    targetDesktop = {
```

Comment:
Wouldn't it make sense to add `desktop` as a valid value here?

Reply:
Not sure what would we gain by adding that extra option?

Reply:
We would gain the ability to specify `target-os` as `mobile` instead of any specific target if in that context it makes sense. I'm trying to cover a potential use, but it's also just aesthetics. If the flag is called `targetDesktop`, it should be triggerable via `desktop` option, and so on. It's just intuitive in my opinion. But this is just a tiny nitpick.